apps/static/base.css

```
@@ -156,43 +250,235 @@ body {
+.dark .text-primary-emphasis { color: #9ec0ff; }
+
+/* ===== Visually hidden / screen reader ===== */
+.visually-hidden,
+.sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)